SECURITY.rst

```diff
@@ -0,0 +1,27 @@
+.. RERO ILS Copyright (C) 2022 RERO
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, version 3 of the License.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+Security policy
+===============
+
+If you wish to disclose a security issue occurring on this projet,
+please send an e-mail to the following address:
+
+`security@rero.ch`_
```

Comment:
Why _

Reply:
Don't know why. It is like this in rero-ils, some sort of rst test formating.